v3/lints/cabf_br/lint_sub_ca_eku_incompatible_values_test.go

```diff
@@ -0,0 +1,67 @@
+package cabf_br
+
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"testing"
+
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/test"
+)
+
+func TestSubCAEkuServerAndClient(t *testing.T) {
+	inputPath := "SubCAEkuServerAndClient.pem"
+	expected := lint.Pass
+	out := test.TestLint("e_sub_ca_eku_incompatible_values", inputPath)
+	if out.Status != expected {
+		t.Errorf("%s: expected %s, got %s", inputPath, expected, out.Status)
+	}
+}
+
+func TestSubCAEkuServerAndCode(t *testing.T) {
+	inputPath := "SubCAEkuServerAndCode.pem"
+	expected := lint.Error
+	out := test.TestLint("e_sub_ca_eku_incompatible_values", inputPath)
+	if out.Status != expected {
+		t.Errorf("%s: expected %s, got %s", inputPath, expected, out.Status)
+	}
+}
+
+func TestSubCAEkuServerAndTime(t *testing.T) {
+	inputPath := "SubCAEkuServerAndTime.pem"
+	expected := lint.Error
+	out := test.TestLint("e_sub_ca_eku_incompatible_values", inputPath)
+	if out.Status != expected {
+		t.Errorf("%s: expected %s, got %s", inputPath, expected, out.Status)
+	}
+}
+
+func TestSubCAEkuServerAndEmail(t *testing.T) {
+	inputPath := "SubCAEkuServerAndEmail.pem"
+	expected := lint.Error
+	out := test.TestLint("e_sub_ca_eku_incompatible_values", inputPath)
+	if out.Status != expected {
+		t.Errorf("%s: expected %s, got %s", inputPath, expected, out.Status)
+	}
+}
+
+func TestSubCAEkuServerAndAny(t *testing.T) {
+	inputPath := "SubCAEkuServerAndAny.pem"
+	expected := lint.Error
+	out := test.TestLint("e_sub_ca_eku_incompatible_values", inputPath)
+	if out.Status != expected {
+		t.Errorf("%s: expected %s, got %s", inputPath, expected, out.Status)
+	}
+}
```

Comment:
Note: You don't have any tests where EKU is present, but serverAuth is **not** present, while other combinations are. For example, e-mail and code-signing.